### PR TITLE
Remove duplicated openSUSE CPE UNIX family test

### PR DESCRIPTION
**Pull request by abergmann**
The UNIX family test `test_opensuse_unix_family` in `installed_OS_is_opensuse.xml` is duplicated. The patch will reference the generic `test_unix_family` from `installed_OS_is_part_of_Unix_family.xml`.

### CODE REVIEW

File: shared/checks/oval/installed_OS_is_opensuse.xml [hunk 1]
```diff
@@ -14,20 +14,11 @@
     </metadata>
     <criteria>
       <criterion comment="Installed operating system is part of the unix family"
-      test_ref="test_opensuse_unix_family" />
+      test_ref="test_unix_family" />
```
Comment (redhatrises) [inline shared/checks/oval/installed_OS_is_opensuse.xml]: @abergmann this change is incorrect. Please change it to:
```<extend_definition comment="Installed OS is part of the Unix family" definition_ref="installed_OS_is_part_of_Unix_family" />```

File: opensuse/cpe/opensuse-cpe-dictionary.xml [hunk 2]
```diff
@@ -2,19 +2,24 @@
...
+      <cpe-item name="cpe:/o:opensuse">
+            <title xml:lang="en-us">openSUSE</title>
+            <!-- the check references an OVAL file that contains an inventory definition -->
+            <check system="http://oval.mitre.org/XMLSchema/oval-definitions-5" href="filename">installed_OS_is_opensuse</check>
+      </cpe-item>
       <cpe-item name="cpe:/o:opensuse:leap:42.1">
             <title xml:lang="en-us">openSUSE Leap 42.1</title>
             <!-- the check references an OVAL file that contains an inventory definition -->
-            <check system="http://oval.mitre.org/XMLSchema/oval-definitions-5" href="filename">installed_OS_is_opensuse</check>
+            <check system="http://oval.mitre.org/XMLSchema/oval-definitions-5" href="filename">installed_OS_is_opensuse421</check>
```
Comment (redhatrises) [inline opensuse/cpe/opensuse-cpe-dictionary.xml]: @abergmann why this change? This is adds a lot of maintenance for upstream activities. In addition, the change doesn't really need to happen if you are checking for any version of openSUSE 42

**Comment by redhatrises** [comment]
@abergmann please fix the changes. Some of the changes I don't think are necessary as in breaking out checks for specific versions of upstream releases.

**Comment by redhatrises** [comment]
@abergmann sorry but do you mind removing commit a464659 that was added to this PR?

**Comment by abergmann** [comment]
Sorry for the noise. I had some problems to remove the unwanted commit and ended up by closing this PR. I will try again.

**Comment by jan-cerny** [comment]
@abergmann The magic command is `git rebase -i upstream/master` (assuming `upstream` is `git@github.com:OpenSCAP/scap-security-guide.git`).
